core/templates/dev/head/domain/skill/ConceptCardBackendApiService.ts

```diff
@@ -48,69 +48,83 @@ oppia.factory('ConceptCardBackendApiService', [
       });
     };
 
-    var _isCached = function(skillId) {
-      return _conceptCardCache.hasOwnProperty(skillId);
+    var _isCached = function(skillIds) {
```

Comment:
I don't think this question was answered - why are we checking if the entire list of skillIDs is cached? It will fail anytime even one of the skillIds is uncached, and we will end up doing unnecessary fetches.
We should be checking isCached per skill ID

Reply:
Checking one skillID instead now.